local_planner/src/nodes/local_planner_node.cpp

```diff
@@ -64,33 +64,10 @@ LocalPlannerNode::LocalPlannerNode(const bool tf_spin_thread) {
   px4_param_sub_ = nh_.subscribe("/mavros/param/param_value", 1,
                                  &LocalPlannerNode::px4ParamsCallback, this);
 
+  // initialize standard publishers
   world_pub_ = nh_.advertise<visualization_msgs::MarkerArray>("/world", 1);
   drone_pub_ = nh_.advertise<visualization_msgs::Marker>("/drone", 1);
```

Comment:
Why are these two not moved into the visualizer?

Reply:
because they are not local_planner specific. I wanted to have a node which does all the general stuf input/output to sensors and fcu and general SITL stuff. The visualizer is supposed to do all the local_planner specific visualization we need for debugging. Such that in far futur we could swap the local_planner with e.g. an ompl planner, reuse the same node with a different visualizer. But if those publishers annoy you where they are we can discuss putting the in the visualizer also, then we you just have to duplicate them for all visualizers (or have a base class visualizer, where all planners share this one)